docs/machine-learning/tutorials/predict-prices-with-model-builder.md

```diff
@@ -0,0 +1,195 @@
+---
+title: 'Predict prices using regression with Model Builder'
+description: This tutorial illustrates how to build a regression model using ML.NET Model Builder to predict prices, specifically, New York City taxi fares.
+author: luisquintanilla
+ms.author: luquinta
+ms.date: 06/26/2019
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a non-developer, I want to use Model Builder to automatically generate a model to predict prices using Model Builder. 
+---
+
+# Predict prices using regression with Model Builder
+
+This tutorial illustrates how to build a [regression model](../resources/glossary.md#regression) using ML.NET Model Builder to predict prices, specifically, New York City taxi fares. The regression machine learning task can be utilized in scenarios where the target or label to predict is numerical.
+
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+> * Prepare and understand the data
+> * Choose a scenario
+> * Load the data
+> * Train the model
+> * Evaluate the model
+> * Use the model for predictions
+
+> [!NOTE]
+> Model Builder is currently in Preview.
+
+## Pre-requisites
+
+For a list of pre-requisites and installation instructions, visit the [Model Builder installation guide](../how-to-guides/install-model-builder.md).
+
+## Create a console application
+
+1. Create a **.NET Core Console Application** called "TaxiFarePrediction".
+
+1. Create a directory named *Data* in your project to store the data set files.
```

Comment:
Would the steps be clearer if we created the Data directory at the same time as we download the data?

Reply:
#resolved